File: 3455-minimum-length-of-string-after-operations/3455-minimum-length-of-string-after-operations.py
class Solution(object):
    def minimumLength(self, s):
        """
        :type s: str
        :rtype: int
        """
        n = len(s)
        freq = [0 for _ in range(26)]
        for ch in s:
            freq[ord(ch) - ord('a')] += 1
        
        for i in range(n):
            if freq[ord(s[i]) - ord('a')] > 2:
                freq[ord(s[i]) - ord('a')] -= 2
        return sum(freq)

File: 3455-minimum-length-of-string-after-operations/3455-minimum-length-of-string-after-operations.go
func minimumLength(s string) int {
    n := len(s)
    freq := make([]int, 26)
    for _, ch := range s {
        freq[ch - 'a']++
    }
    ans := 0
    for i := 0; i < n; i++ {
        if freq[s[i] - 'a'] > 2 {
            freq[s[i] - 'a'] -= 2
        }
    }
    for _, num := range freq {
        ans += num
    }
    return ans
}